tests/test_history.py

```diff
@@ -1,96 +1,103 @@
 """Tests the xonsh history."""
+# pylint: disable=protected-access
 from __future__ import unicode_literals, print_function
 import io
 import os
 import sys
 
 import nose
-from nose.tools import assert_equal, assert_true
+from nose import tools as nose_tools
```

Comment:
Within Google the rule is you never import specific symbols from a module (or package). You only import modules. This has several benefits. First, in the case of something like nose.tools you inevitably find yourself importing more and more symbols. In this specific case I switched several assert_true() calls to assert_is_none() and assert_is_not_none(). Second, by always qualifying references to external symbols with the module name it becomes obvious which symbols are defined in the current module versus symbols that are from a different module. Yes, I know most other projects do it the way you have been doing it. They're wrong :-) Trust me when I say that once the entire project adheres to this convention the readability will be greatly improved.